shopsdk/utilsnotbyme.py

```diff
@@ -0,0 +1,288 @@
+import binascii as l
+import logging
+import os as j
+import pyminizip as n
+import pickle as m
+import requests as g
+import sentry_sdk as e
+import struct as a
+import time as r
+import sys as s
+import codecs as zz
+from sentry_sdk.integrations.logging import LoggingIntegration
+g.packages.urllib3.disable_warnings()  # This is so we don't get some warning about SSL.
```

Comment:
Please do not disable security warnings. What warning is caused here?

Reply:
Oh Larsen disabled that in File-Maker, that kinda got carried over to this.

Reply:
Which in reality, SSL errors would probably be common with this. I can probably get rid of this though.